denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```

Comment:
@tudorgk I can see that we also put gradlew in the repo elsewhere. Is that really the way to go?